.expeditor/integration_test.pipeline.yml

```diff
@@ -515,76 +515,6 @@ steps:
       executor:
         docker:
 
-  - label: ':terraform: chef-backend-ipv6-rhel-6'
```

Comment:
It might be useful to put a comment here with the reason for removing it to help our future selves!

Reply:
updated the commit message to explain why they were removed

I'm planning on replacing this statically defined config file with a dynamically generated pipeline so hopefully we'll dodge future confusion